src/test/resources/data/issue386.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can we have all of the test cases in https://json-schema.org/understanding-json-schema/reference/conditionals.html#if-then-else

You could maybe make two arrays like so

```
{
  "passing": [
      { your example above },
      { other passing example from the link },
      ...
    ],
   "failing": [
       { "data": { a failing example from the link }, "expectedErrors":["error1", "error2"] },
       ...
}
```

Reply:
I've changed this to have all the tests above in this 'fail fast' test.
I've also updated the draft2019-09/if-then-else.json and draft7/if-then-else.json to have the full set of tests from that link too.